CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Dev: Add warning logs for improper runescape settings that impact notifiers. (#92)
```

Comment:
Should probably be changed to the `Minor` category now that it has  user-visible impact